scripts/insert_access_token.sh

```diff
@@ -1,12 +1,15 @@
+#!/bin/bash
+
 echo "Inserting Mapbox access token..."
 token_file=~/.mapbox
 token_file2=~/mapbox
 token="$(cat $token_file 2>/dev/null || cat $token_file2 2>/dev/null)"
+plist="$TARGET_BUILD_DIR/$INFOPLIST_PATH"
 if [ "$token" ]; then
-  plutil -replace MBXAccessToken -string $token "$TARGET_BUILD_DIR/$INFOPLIST_PATH"
+  plutil -replace MBXAccessToken -string "$token" "$plist"
   echo "Token insertion successful"
-else
+elif plutil -extract MBXAccessToken xml1 "$plist" -o - | grep "<string></string>"; then
```

Comment:
Would you mind to use `/usr/libexec/PlistBuddy -c "Print : MBXAccessToken" "$plist"` here? It would return a clean token or non-0 exit code if no entry was found.

Reply:
Should we also extract the `MBXAccessToken` constant to the variable? 🤔

Reply:
Updated